src/brain/modelgen/LSTM_Model.py

```diff
@@ -0,0 +1,133 @@
+import tensorflow as tf
```

Comment:
I think we need copyright header for these model files as well.

Reply:
The `formatter.py` script cant create a copyright header for Python files currently. Should I add in the functionality?

Reply:
Then you need to be watchful to only create header for py files in src dir. You don’t want to create headers for all the scripts in the repo.